lithops/standalone/standalone.py

```diff
@@ -244,6 +235,28 @@ def dismantle(self):
         """
         self.backend.stop()
 
+    def init(self):
+        """
+        Start the VM instance and initialize runtime
+        """
+        # if not started, start the vm
+        if not self._is_backend_ready():
+            # The VM instance is stopped
+            init_time = time.time()
+            self.backend.start()
+            self._wait_backend_ready()
+            total_start_time = round(time.time()-init_time, 2)
+            logger.info('VM instance ready in {} seconds'.format(total_start_time))
+
+        # Not sure if mandatory, but sleep several seconds to let proxy server start
+        time.sleep(2)
+
+        # if proxy not started, install it
+        if not self._is_proxy_ready():
+            self._setup_proxy()
```

Comment:
@kpavel I understand what you want to do, but this logic will prevent remote lithops version being updated. For example, let's say you have lithops 2.2.2 installed in your laptop and in your VM, and then update to 2.2.3. The VM will have the proxy installed for 2.2.2 so it will never call self. _setup_proxy(), so the VM will always run 2.2.2

Reply:
You are right. Fixed it by moving the backend start logic to separate method.